scipy/stats/_axis_nan_policy.py

```diff
@@ -74,7 +74,7 @@ def _broadcast_concatenate(xs, axis):
 
 # TODO: add support for `axis` tuples
 def _remove_nans(samples, paired):
```

Comment:
This function is exactly equivalent to `_remove_sentinel(samples, paired, sentinel=np.nan)`. So, we can remove the repeated code and just `_remove_sentinel(samples, paired, sentinel=np.nan)`.

Reply:
Yes, there's a comment about that in the code. I know how to consolidate them, but chose not to right now.